e2e/fixtures/events/cui/defendantResponseCui.js

```diff
@@ -299,8 +299,8 @@ module.exports = {
           reasonForHearingAtSpecificCourt: 'court',
           responseCourtLocations: [],
           caseLocation: {
-            region: 'Barnet Civil and Family Centre - St Mary\'s Court, Regents Park Road - N3 1BQ',
-            baseLocation: 'Barnet Civil and Family Centre - St Mary\'s Court, Regents Park Road - N3 1BQ'
+            region: 'Central London County Court - Thomas More Building, Royal Courts of Justice, Strand, London - WC2A 2LL',
```

Comment:
ideally it should be used from config.claimantSelectedCourt / config.defendantSelectedCourt so that it will be easy to manage :)